src/scripts/main.js

```diff
@@ -1 +1,9 @@
 'use strict';
+
+const a = document.querySelector('#thumbs');
```

Comment:
```suggestion
const list = document.querySelector('#thumbs');
```
Это список, а не ссылка